test/test_indexing.py

```diff
@@ -142,6 +142,20 @@ def test_same_expt_name(session_db):
     assert(len(r) == 2)
     assert(r[0].root_dir != r[1].root_dir)
 
+def test_following_symlinks(session_db):
+    session, db = session_db
+
+    # Indexing symlinked experiment should fail with default arguments
+    database.build_index('test/data/indexing/symlinked/experiment_a', session)
```

Comment:
The required test files don't seem to exist? (Also Travis looks to be broken, I'll take a look at that)

Reply:
I forgot to add them. Tah.